optuna/logging.py

```diff
@@ -102,6 +102,31 @@ def get_verbosity() -> int:
 def set_verbosity(verbosity: int) -> None:
     """Set the level for the Optuna's root logger.
 
+    Example:
+
+        Set the logging level ``optuna.logging.INFO``.
```

Comment:
Since using `INFO` level  shows no difference from the default behavior, how about setting the level to `WARN` or `ERROR`?